### PR TITLE
Update monensin equation

### LINKED ISSUE #1493: Equation for monensin awaiting updates
Dr. Edward Garcia recently updated a new set of equation for the mitigation effect of monensin. The codebase and design doc need to be updated.   

## Expected Behavior

The codebase and design doc are up to date according to the recent version of the monensin equation that Edward came up with. 

## Current Behavior

The current codebase and design doc is outdated. 

## Possible Solution

Update the codebase and design doc according to the recent updates by Edward.  

## Steps to Reproduce

1.
2.
3.
4.

### PR DESCRIPTION
**Pull request by tomhuhh**
### Context

Issue(s) closed by this pull request: closes #1493 

Dr. Edward Garcia recently updated a new set of equation for the mitigation effect of monensin. The codebase and design doc need to be updated.

- [x] The [changelog](https://docs.google.com/spreadsheets/d/1U-3igxdstHTFb6k5dVcE5-x9IL0r6q18eUiwwytKZyA/edit#gid=0) has been updated.

### What

This PR updates:  
1. the methane yield mitigation equation based on the updates from Edward
2. the metadata file for options for methane mitigation feed additive  

It is worth noting that the original equation by Edward does not provide lower or upper bounds of the dietary starch content for the equation to work. Caution should be taken when applying it to a variety of diets. You may see an increase in lactating cow methane emission if the starch content is too low. 

The updated design doc is available here: https://docs.google.com/document/d/12GsTsUlm4TzIVOo05pRpsaN-TrkjHg5KGO5MOAz8DOY/edit?usp=sharing.  

### Test plan

I would recommend uploading a copy of regional diets and the feed library from the pilot testing branch to your local to test the mitigation impact of monensin.

### CODE REVIEW

File: RUFAS/routines/animal/manure/lactating_cow_manure_excretion.py [hunk 1]
```diff
@@ -45,8 +42,6 @@ def methane_mitigation(
     """
 
     methane_yield_reduction = 0.0
-    Monensin_CP_lower_bound = AnimalModuleConstants.MONENSIN_CP_LOWER_BOUND
-    Monensin_CP_upper_bound = AnimalModuleConstants.MONENSIN_CP_UPPER_BOUND
```
Comment (ltp23) [inline RUFAS/routines/animal/manure/lactating_cow_manure_excretion.py:48]: Should we remove these constants in the `AnimalModuleConstants` class also?

File: input/metadata/properties/default.json [hunk 2]
```diff
@@ -713,7 +713,7 @@
         "type": "string",
         "description": "Methane Mitigation Method (select one) -- The type of enteric methane mitigation supplement that is fed to the lactating cows. Note that default values given are not implemented unless the specific supplement is selected here",
         "default": "None",
-        "pattern": "^(None|3-NOP|monensin|essential_oils|seaweed)$"
+        "pattern": "^(None|3-NOP|Monensin|Essential Oils|Seaweed)$"
```
Comment (ltp23) [inline input/metadata/properties/default.json:716]: This has been uncaught probably because we have used the `None` and `3-NOP` options only so far.

**Review by ltp23 — APPROVED** [review]
Looks good to me! Thanks Haowen.

Can you also add docstring descriptions for these two parameters `methane_mitigation_method` and `methane_mitigation_additive_amount` in the `manure_calculations` method?

**Comment by tomhuhh** [comment]
Thanks for your feedback on this Loi! I've addressed your comments. Please take another look when you have a moment. 

A note for future: currently the code only applies methane mitigation feed additives for `lactating cows`. If we are interested in the mitigation impacts on other animal categories, new equations need to be developed and/or implemented.

**Review by KFosterReed — APPROVED** [review]
This works as expected! Well done Haowen!

First figure is without monensin in the FARMES Default NE diet, second is with!
image

image

**Comment by ltp23** [comment]
The new updates look good to me. Thanks Haowen!

**Pull request merged**